pkg/keyidentifier/helpers.go

```diff
@@ -0,0 +1,51 @@
+package keyidentifier
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/pem"
+	"io"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func truePtr() *bool {
```

Comment:
IMO these helpers are excessive

Reply:
Yeah. you're probably right. I need the boolPtr, unless I ditch pointers. But I can ditch the truePtr and falsePtr